### PR TITLE
T2T2 project SDK adapter

### PR DESCRIPTION
**Pull request by Holly-S33**
##### Name (to be shown on DefiLlama):
T2T2

##### Twitter Link:
https://twitter.com/T2T2_official

##### List of audit links if any:
https://sharkteam.org/report/audit/20231104002C_en.pdf

##### Website Link:
https://www.t2click.com

##### Logo (High resolution, will be shown with rounded borders):
https://t2-token-pool.t2t2.com/t2t2_token_icon.png

##### Chain:
Ethereum + Base

##### Short Description (to be shown on DefiLlama):
T2T2 is a social trading platform powered by social bonding curve and fund investing curve.

##### Token address and ticker if any:
0x390e61F798267fe7aa9BBE61Be8BB1776250D44C

##### Category (full list at https://defillama.com/categories) *Please choose only one:
SoFi

##### Github org/user (Optional, if your code is open source, we can track activity):

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/T2T2 exports TVL: 
        
 
 
    ethereum                  0
    base                      24.35 k
    
    total                    24.35 k

File: projects/T2T2/index.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const { nullAddress, sumTokensExport } = require("../helper/unwrapLPs");
+
+const config = {
+  ethereum: '0x390e61F798267fe7aa9BBE61Be8BB1776250D44C',
```
Comment (g1nt0ki) [inline projects/T2T2/index.js:4]: hi @Holly-S33, thanks for the PR
the base contract looks right, but Ethereum one is a token contract?

Comment (Holly-S33) [reply projects/T2T2/index.js]: Sorry is my mistake. this is the wrong address, I have removed the wrong address

**Comment by llamatester** [comment]
The adapter at projects/T2T2 exports TVL: 
        
 
 
    base                      26.56 k
    
    total                    26.56 k

**Pull request merged**